### DISCUSSION TITLE
When will @tauri-apps/api be published?

### DISCUSSION THREAD

**Discussion by LIMPIX31** [tauri-apps/tauri#5073] [Q&A] [answered]
When will `@tauri-apps/api` be published?
I've been working on improving `@tauri-apps/api` and my pull request was accepted. However, the last publication was 2 months ago. How can I use my `@tauri-apps/api` changes in a project now?

**Comment by FabianLars** [comment, marked as answer]
> When will @tauri-apps/api be published?

When tauri v1.1 is ready. Some last open PRs/issues here and there, and the (internal-ish) audit left (afaik), so shouldn't be too far in the future.
On that note, yes we want to/will have a more rapid release schedule, but that needs a solid audit pipeline in place...

> How can I use my @tauri-apps/api changes in a project now?

https://tauri.app/v1/guides/faq#how-can-i-use-unpublished-tauri-changes
On top of that you could use `yarn pack` to create a bundle which you can check-in to your repo and depend on in package.json.
